collated/main/A0141128R.md

```diff
@@ -0,0 +1,692 @@
+    		return true;
+    	else
+    		return false;
+    }
+
+    @Override
+    public int hashCode() {
+    	return Objects.hash(start, end);
+    }
+
+```
+###### /java/seedu/cmdo/model/task/Priority.java
+``` java
+    //getter to get value 
+    public String getValue(){
+    	return value;
+    }
+    //setter for priority
+    public void setPriority(String value){
+    	this.value = value;
+    }
+
+    @Override
+    public String toString() {
+        return value;
+    }
+    /*
+    @Override
+    public boolean equals(Object other) {
+        return other == this // short circuit if same object
+                || (other instanceof Priority // instanceof handles nulls
+                && this.value.equals(((Priority) other).value)); // state check
+    }
+    */
+
+    @Override
+    public int hashCode() {
+        return value.hashCode();
+    }
+
+}
+```
+###### /java/seedu/cmdo/model/task/Task.java
+``` java
+    //to edit it to a floating task
+    public void setFloating(){
+    	dueByTime.setFloating();
+    	dueByDate.setFloating();
+	}
+    
+    //change detail used in edit command
+    public void setDetail(Detail d) {
+        detail = d;
+    }
+    //setter to edit due by date for edit command
+    public void setDueByDate(DueByDate dbd){
+    	dueByDate = dbd;
+    }
+    //setter to edit due by time for edit command
+    public void setDueByTime(DueByTime dbt){
+    	dueByTime = dbt;
+    }
+    
+	@Override
+    public Detail getDetail() {
+        return detail;
+    }
+
+    @Override
+    public Done checkDone() {
+        return done;
+    }
+    
+    @Override
+    public DueByDate getDueByDate() {
+        return dueByDate;
+    }
+
+    @Override
+    public DueByTime getDueByTime() {
+        return dueByTime;
+    }
+    
+    @Override
+    public Priority getPriority() {
+        return priority;
+    }
+
+    @Override
+    public UniqueTagList getTags() {
+        return new UniqueTagList(tags);
+    }
+    
+    public void updateObjectID() {
+        objectID = UUID.randomUUID().toString();
+    }
+
+    /**
+     * Replaces this task's tags with the tags in the argument tag list.
+     */
+    public void setTags(UniqueTagList replacement) {
+        tags.setTags(replacement);
+    }
+
+    @Override
+    public boolean equals(Object other) {
+        return other == this // short circuit if same object
+                || (other instanceof ReadOnlyTask // instanceof handles nulls
+                && this.isSameStateAs((ReadOnlyTask) other));
+    }
+
+    @Override
+    public int hashCode() {
+        // use this method for custom fields hashing instead of implementing your own
+        return Objects.hash(detail, done, dueByDate, dueByTime, priority, tags);
+    }
+
+    @Override
+    public String toString() {
+        return getAsText();
+    }
+    
+    /**
+     * Marks a floating task via boolean
+     * 
+     * @author A0141006B
```

Comment:
why are codes under others here? 
if it is just setters and getters that they have written, they can don't claim it, it doesn't really add much value to theri collated codes please sort this out amongst urselves,